src/components/Header/index.tsx

```diff
@@ -0,0 +1,15 @@
+import "~/assets/tailwind.css";
+import { A } from "@solidjs/router";
+
+export default function Header(props: any) {
+	const containerStyles = 'flex items-center text-[var(--specialTextColor)] mb-4 px-1';
```

Comment:
you know you can create a custom class in tailwind so you can just use `text-special` and it will call that variable. The arbitrary values can be hard to manage sometimes!